jupyter_server/serverapp.py

```diff
@@ -1604,31 +1604,12 @@ def init_httpserver(self):
 
     @staticmethod
     def _init_asyncio_patch():
-        """set default asyncio policy to be compatible with tornado
-        Tornado 6 (at least) is not compatible with the default
-        asyncio implementation on Windows
-        Pick the older SelectorEventLoopPolicy on Windows
-        if the known-incompatible default policy is in use.
-        do this as early as possible to make it a low priority and overrideable
-        ref: https://github.com/tornadoweb/tornado/issues/2608
-        FIXME: if/when tornado supports the defaults in asyncio,
-               remove and bump tornado requirement for py38
-        """
-        if sys.platform.startswith("win") and sys.version_info >= (3, 8):
-            import asyncio
-            try:
-                from asyncio import (
-                    WindowsProactorEventLoopPolicy,
-                    WindowsSelectorEventLoopPolicy,
-                )
-            except ImportError:
-                pass
-                # not affected
-            else:
-                if type(asyncio.get_event_loop_policy()) is WindowsProactorEventLoopPolicy:
-                    # WindowsProactorEventLoopPolicy is not compatible with tornado 6
-                    # fallback to the pre-3.8 default of Selector
-                    asyncio.set_event_loop_policy(WindowsSelectorEventLoopPolicy())
+        """no longer needed with tornado 6.1"""
+        warnings.warn(
```

Comment:
Should this remain behind a windows-3.8 check or be removed altogether?  I understand its purpose, but now folks not on Windows-py3.8 systems will get this warning when it never applied to them in the first place.

Reply:
removed entirely... but while hidden behind a "private" member, I wouldn't rule out people having used it to avoid The Condition. Could just be removed immediately, but a release with a warning seems reasonable vs  Your Stuff Breaks Now.

Reply:
Right on.  I missed that the call server makes has been removed.  This makes perfect sense - thanks.